waspc/src/Wasp/Generator/ServerGenerator.hs

```diff
@@ -115,8 +117,11 @@ npmDepsForWasp spec =
             ("jsonwebtoken", "^8.5.1"),
             ("secure-password", "^4.0.0"),
             ("dotenv", "8.2.0"),
-            ("helmet", "^4.6.0")
+            ("helmet", "^4.6.0"),
+            -- NOTE: Not in DevDeps so we can patch in prod contexts (e.g., Docker, PaaS, etc.)
```

Comment:
Well it makes sense! It is not used just for development -> it is part of the source of the program. I would probably even skip this comment, assuming it is clear to the reader.

Reply:
Ok, I can nix the comment, thnx.